redisinsight/ui/src/components/consents-settings/ConsentOption/components/ItemDescription.tsx

```diff
@@ -1,26 +1,29 @@
-import { EuiLink } from '@elastic/eui'
 import parse from 'html-react-parser'
 import React from 'react'
+import { Link } from 'uiSrc/components/base/link/Link'
 
 interface ItemDescriptionProps {
-  description: string;
-  withLink: boolean;
+  description: string
+  withLink: boolean
 }
 
-export const ItemDescription = ({ description, withLink }: ItemDescriptionProps) => (
-    <>
-      {description && parse(description)}
-      {withLink && (
-        <>
-          <EuiLink
-            external={false}
-            target="_blank"
-            href="https://redis.io/legal/privacy-policy/?utm_source=redisinsight&utm_medium=app&utm_campaign=telemetry"
-          >
-            Privacy Policy
-          </EuiLink>
-          .
-        </>
-      )}
-    </>
-)
\ No newline at end of file
+export const ItemDescription = ({
+  description,
+  withLink,
+}: ItemDescriptionProps) => (
+  <>
+    {description && parse(description)}
+    {withLink && (
+      <>
+        <Link
+          color="primary"
+          target="_blank"
+          href="https://redis.io/legal/privacy-policy/?utm_source=redisinsight&utm_medium=app&utm_campaign=telemetry"
```

Comment:
Tip: It's not related to your visual changes at all, but if you want, you can move this constant to live with the rest `EXTERNAL_LINKS` [here](https://github.com/redis/RedisInsight/blob/293cd8f0ad23a1cff64ea425ec1e7a3b1cf3bf9e/redisinsight/ui/src/constants/links.ts).

And then use the following helper to build up the URL, instead of having all the query params hardcoded:

```suggestion
<Link
    color="primary"
    target="_blank"
    href={getUtmExternalLink(EXTERNAL_LINKS.legalPrivacyPolicy, {
        medium: UTM_MEDIUMS.App,
        campaign: 'telemetry',
    })}
>
    Privacy Policy
</Link>
```